.github/actions/build-core/action.yml

```
@@ -7,6 +7,6 @@ runs:
       shell: bash
       run: |
         git clone https://github.com/trustyai-explainability/trustyai-explainability.git
-        mvn clean install -DskipTests -f trustyai-explainability/pom.xml -Pshaded -fae -e -nsu
-        mv trustyai-explainability/explainability-core/target/explainability-core-*-SNAPSHOT.jar src/trustyai/dep/org/trustyai/
+        mvn clean install -DskipTests -f trustyai-explainability/explainability-arrows/pom.xml -Pshaded -fae -e -nsu
```

Please continue to build from root so that parent is compiled too
```suggestion
        mvn clean install -DskipTests -f trustyai-explainability/pom.xml -Pshaded -fae -e -nsu
```